bin/notification.js

```diff
@@ -24,25 +24,20 @@ werelogs.configure({
 /* eslint-disable no-param-reassign */
 function queueBatch(queuePopulator, taskState) {
     if (taskState.batchInProgress) {
-        log.warn('skipping notification batch: previous one still in progress');
+        log.debug('skipping notification batch: previous one still in progress');
         return undefined;
     }
     log.debug('start queueing notification batch');
     taskState.batchInProgress = true;
     const maxRead = qpConfig.batchMaxRead;
-    queuePopulator.processAllLogEntries({ maxRead }, (err, counters) => {
+    queuePopulator.processAllLogEntries({ maxRead }, err => {
         taskState.batchInProgress = false;
         if (err) {
             log.error('an error occurred during notification', {
                 method: 'QueuePopulator::task.queueBatch',
                 error: err,
             });
-            return undefined;
         }
-        const logFunc = (counters.some(counter => counter.readRecords > 0) ?
-            log.info : log.debug).bind(log);
-        logFunc('notification batch finished', { counters });
```

Comment:
Is this log being shown somewhere else or are we completely removing the log?

Reply:
It is being shown on log reader. Introduced in https://github.com/scality/backbeat/pull/1234/files#diff-ffc77ab2f45340e1e9848a1814fb0040f0457ef4b53c851288313c7f6b213dd1R226 .. though it says `replication batch finished`..should be generic I think

Reply:
Ok please change that